doc/setup.md

```diff
@@ -55,8 +55,19 @@ To use **N-API** in a native module:
 ```gyp
   'defines': [ 'NAPI_DISABLE_CPP_EXCEPTIONS' ],
 ```
-
-  4. Include `napi.h` in the native module code.
+  4. **Only** for **macOS** user please don't forget to insert these additional 
```

Comment:
Perhaps you should rephrase this as "If you would like your native addon to support OSX, please also add the following:"